### ISSUE TITLE
GLIBCXX_3.4.29 not found - AWS Platform Node.js 18 running on 64bit Amazon Linux 2/5.8.2

### ISSUE THREAD

**Issue by ikb42** [node-ffi-napi/node-ffi-napi#252]
Hi
My app runs fine under AWS Platform Node.js 16 but under 18 I get this error when the app starts up:

> /lib64/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /var/app/current/node_modules/ffi-napi/build/Release/ffi_bindings.node

Running this on the instance gives:
`strings /usr/lib64/libstdc++.so.6 | grep GLIBCXX_3.4`

```GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBCXX_3.4.20
GLIBCXX_3.4.21
GLIBCXX_3.4.22
GLIBCXX_3.4.23
GLIBCXX_3.4.24
```

Help much appreciated, thanks

**Comment by ikb42** [comment]
Update on this - TL;DR it works now!

AWS have released their EB managed Node 18 with Amazon Linux 2023, this package works without modification.
See "Node.js 18 running on 64bit Amazon Linux 2023/6.0.0"

**Comment by cclauss** [comment]
> TL;DR it works now!

Are they using https://github.com/2060-io/node-ffi-napi instead?

Can we prove that in CI?
* #255
* https://ci.appveyor.com/project/addaleax/node-ffi-napi/builds/47874067
* #199
* #253
---
### should throw an Error when invoked after the callback gets garbage collected
```
  Callback
    √ should create a C function pointer from a JS function
    √ should be invokable by an ffi'd ForeignFunction (128ms)
    √ should work with a "void" return type (109ms)
    √ should not call "set()" of a pointer type (151ms)
    √ should throw an Error when invoked through a ForeignFunction and throws (105ms)
    √ should throw an Error with a meaningful message when a type's "set()" throws (97ms)
    - should throw an Error when invoked after the callback gets garbage collected
    √ should propagate callbacks and errors back from native threads (204ms)
    async
      √ should be invokable asynchronously by an ffi'd ForeignFunction (146ms)
```
